go.sh

```diff
@@ -429,6 +429,8 @@ esac
             || fail "Failed to pull WatchTower image '$wt_image' from Docker Hub"
     }
 
+    modinfo wireguards &> /dev/null || rem "Missed WireGuard kernel module"
```

Comment:
/WireGuard/wireguard/ and please add ", protocol is disabled"

Reply:
Done!